### DISCUSSION TITLE
[feature] Always localize places names

### DISCUSSION THREAD

**Discussion by teaalltr** [immich-app/immich#9678] [Feature Request]
### The bug

At the moment, Immich don't always follow the localized place translation.
For instance it should always use "Roma" instead of "Rome" or "Praga" instead of "Prague" when the app language is set to Italian.

### The OS that Immich Server is running on

RaspberryPi OS

### Version of Immich Server

NN

### Version of Immich Mobile App

1.105.1

### Platform with the issue

- [ ] Server
- [ ] Web
- [x] Mobile

### Your docker-compose.yml content

```YAML
NN
```

### Your .env content

```Shell
NN
```

### Reproduction steps

```bash
1.NN
2.
3.
...
```

**Comment by cqdjcc** [comment]
建议支持各种语言

**Comment by DenisPac** [comment]
Hello,
This is still the same situation in current version 1.126.1.
In the recent new Group by Country feature, the names of countries is in English and not in GUI language.
The same applies to the main map, mostly in english, with some local places shown in local language.

**Comment by kwongtn** [comment]
To add on for this, it would be great that if we could specify the languages we would like to see in their actual form instead of the romanized version.

For example I read English, Chinese & Japanese. Hence it would make sense for me to view the places of China, Taiwan and Japan in their local form.

**Comment by KENUNK** [comment]
求增加这个功能，地点名称根据切换的语音更改，切换为中文，地点名称也可以用中文显示，等这个功能已经快半年了，一直没更新

**Comment by jaymi-lai** [comment]
Please help to equip the localized modules, just like immich PC using Hosted Weblate service, and we can help to input translation

**Comment by alexanderek** [comment]
It would be great if Immich could always display place names in the user’s language,
using the localized entries from the GeoNames alternateNames field.
Right now, names are often shown in English or transliterated form, which looks inconsistent for non-English users.
Supporting locale selection (server-wide or per user) would make the gallery feel much more natural.

**Reply by ataraxus** [reply]
is there any workaround you eventually found

**Comment by xqboyyy** [comment]
Has it not been resolved yet?
<img width="408" height="543" alt="image" src="https://github.com/user-attachments/assets/c14f95d8-c35f-4dc3-8f47-484083f83d90" />
